cmd/kube-apiserver/app/server.go

```diff
@@ -284,7 +273,8 @@ func CreateKubeAPIServerConfig(
 	lastErr error,
 ) {
 	var genericConfig *genericapiserver.Config
-	genericConfig, sharedInformers, versionedInformers, insecureServingInfo, serviceResolver, pluginInitializers, admissionPostStartHook, lastErr = BuildGenericConfig(s.ServerRunOptions, proxyTransport)
+	var storageFactory *serverstorage.DefaultStorageFactory
+	genericConfig, sharedInformers, versionedInformers, insecureServingInfo, serviceResolver, pluginInitializers, admissionPostStartHook, storageFactory, lastErr = BuildGenericConfig(s.ServerRunOptions, proxyTransport)
```

Comment:
> As a future step (not this pull), it seems that we should start pulling the logically required pieces of the return from CreateKubeAPIServerConfig into the master.Config so that the master.Config is a logically contained unit.

Can't agree more